src/platform/sdl/main.c

```diff
@@ -321,3 +337,64 @@ static void mSDLDeinit(struct mSDLRenderer* renderer) {
 
 	SDL_Quit();
 }
+
+void setLogger(struct mCore* core) {
+	int fakeBool = 0;
+	bool logToStdout = true;
+	bool logToFile = false;
+
+	if (mCoreConfigGetIntValue(&core->config, "logToStdout", &fakeBool))
+		logToStdout = fakeBool;
+	if (mCoreConfigGetIntValue(&core->config, "logToFile", &fakeBool))
+		logToFile = fakeBool;
+	const char* logFile = mCoreConfigGetValue(&core->config, "logFile");
+	int filterLevels = core->opts.logLevel;
+
+	_setLogger(logToStdout, logToFile, logFile, filterLevels);
+}
+
+void _setLogger(bool logToStdout, bool logToFile, const char* logFile, int filterLevels) {
+	// Assign basic static variables
+	_logToStdout = logToStdout;
+	_logFile = NULL;
+	
+	if(logToFile && logFile)
+		_logFile = VFileOpen(logFile, O_WRONLY | O_CREAT | O_TRUNC);
+
+	// Create the filter
+	mLogFilterInit(&_filter);
+	mLogFilterSet(&_filter, "gba.bios", mLOG_STUB | mLOG_FATAL);
```

Comment:
This line is unneeded; the reason mLOG_STUB was put in there in the Qt frontend was to give users a dialog when an unimplemented BIOS function is hit, but the SDL frontend doesn't have dialogs.

Reply:
Fixed in https://github.com/mgba-emu/mgba/pull/2345/commits/9440ca24e9b66396445955640548bab981801c2d .

(I just removed the STUB part though... Don't know if you wanted the FATAL removed as well)